src/main/java/seedu/utility/Ui.java

```diff
@@ -492,4 +491,15 @@ public void printCurrentCurrency(CurrencyType currency) {
         System.out.println(Messages.CURRENT_CURRENCY_MESSAGE + currency.toString());
         printLine();
     }
+
+    public void printCurrencyTypes(ArrayList<CurrencyType> currencyTypes) {
+        printLine();
+        System.out.println(Messages.AVAILABLE_CURRENCIES_MESSAGE);
+        int i = 0;
```

Comment:
You might want to refactor this part by doing 
`int i = 1;
for (CurrencyType currencyType : currencyTypes) {
    System.out.println(i + ". " + currencyType);
    i++;
}`
instead?

Reply:
Okay got it will change `i = 0` to `i = 1`.